arrow-fx/build.gradle

```diff
@@ -9,6 +9,10 @@ plugins {
     id "ru.vyarus.animalsniffer"
 }
 
+apiValidation {
```

Comment:
Good job @danimontoya :clap: Just a comment. Could `apiValidation` section be located in `build.gradle`?

Reply:
I guess you mean in the root `build.gradle`? I've been playing with it and no, I couldn't =\
so I had to add it to arrow-fx and worked as I was expecting if I would have added it to the root build.gradle

Reply:
you right @rachelcarmena ! just moved it to the root one! thanks!